### DISCUSSION TITLE
How to config devServer with withModuleFederation ?

### DISCUSSION THREAD

**Discussion by thanhnlh** [nrwl/nx#10209] [General]
Hi, I need to turn of webSocketServer = false so is there a way to config devServer in webpack with withModuleFederation ?

devServer: {
allowedHosts: 'all',
webSocketServer: false,   //https://webpack.js.org/configuration/dev-server/#devserverwebsocketserver  
},

**Comment by DriLLFreAK100** [comment]
Hey man, I've been looking for this too just now. Happens that it's documented here https://nx.dev/recipes/other/customize-webpack#module-federation
